gretel/create_synthetic_data_from_csv_or_df/blueprint.ipynb

```diff
@@ -83,7 +83,7 @@
         "dataset_path = 'https://gretel-public-website.s3-us-west-2.amazonaws.com/datasets/healthcare-analytics-vidhya/train_data.csv'\n",
         "nrows = 10000  # We will use this later when generating data\n",
         "training_df = pd.read_csv(dataset_path, nrows=nrows)\n",
-        "training_df.head()"
+        "training_df"
```

Comment:
Is this intentional not to print just a few lines of the training df?

Reply:
Yep, it will print 20 lines by default, and also show you the overall shape of the dataframe (rows/columns), which head does not.